packages/runtime/runtime.js

```diff
@@ -10,6 +10,7 @@ var runtime = (function (exports) {
 
   var Op = Object.prototype;
   var hasOwn = Op.hasOwnProperty;
+  var define = Object.defineProperty || function (obj, key, desc) { obj[key] = desc.value; };
```

Comment:
Check the line 20, There is already a `define` function, is this intentional?

Reply:
I didn't notice it, thank you!
```suggestion
```

Reply:
But they are different. 
The `define` function at line 20 will return a enumerable method.